samples/cext-trey-research-python/teamsapp.local.yml

```diff
@@ -0,0 +1,53 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.2/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.2
+
+additionalMetadata:
+  sampleTag: Copilot-for-M365-Samples:cext-trey-research-python
+
+provision:
+  # Creates a Teams app
+  - uses: teamsApp/create
+    with:
+      # Teams app name
+      name: cexttreyresearch${{APP_NAME_SUFFIX}}
+    # Write the information of created resources into environment file for
+    # the specified environment variable(s).
+    writeToEnvironmentFile: 
+      teamsAppId: TEAMS_APP_ID
+
+  # Validate using manifest schema
+  #- uses: teamsApp/validateManifest
+    #with:
+      # Path to manifest template
+      #manifestPath: ./appManifest/manifest.json
+
+  # Build Teams app package with latest env value
+  - uses: teamsApp/zipAppPackage
+    with:
+      # Path to manifest template
+      manifestPath: ./appManifest/manifest.json
+      outputZipPath: ./appManifest/build/appManifest.${{TEAMSFX_ENV}}.zip
+      outputJsonPath: ./appManifest/build/manifest.${{TEAMSFX_ENV}}.json
+  # Validate app package using validation rules
+  #- uses: teamsApp/validateAppPackage
+    #with:
+      # Relative path to this file. This is the path for built zip file.
+      #appPackagePath: ./appManifest/build/appManifest.${{TEAMSFX_ENV}}.zip
```

Comment:
Let's uncomment this action

Reply:
Since our manifest shows an error in copilotExtensions, we need to comment out the validation part. We did this by following the example in [cext-trey-research-ts](https://github.com/Jegadeesh-MSFT/Copilot-for-M365-Samples/blob/v-jegadeesh/trey-sample-csharp/samples/cext-trey-research/teamsapp.yml#L51).